.github/workflows/publish-extension.yaml

```diff
@@ -61,6 +61,14 @@ jobs:
           restrict-to: kiwix/kiwix-js
           registries: docker.io
```

Comment:
Shouldn't this line be changed to ghcr.io, following what @kelson42 was asking for?

Reply:
Yes, @kelson42's comment followed the PR. Let me change it and add the credentials as well